lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -134,6 +134,70 @@
       }
     }
   }
+  
+    .btn-blue {
+        @extend %btn-primary-blue;
+        margin-bottom: 0;
+        padding: ($baseline/2.5) ($baseline/2);
+        text-shadow: none;
+    }
+
+    // Custom File upload
+    .customBrowseBtn {
+        display: inline-block;
+        position: relative;
+        margin: ($baseline/2) 0 0;
+        width: 350px;
+        overflow: hidden;
+        vertical-align: bottom;
+
+        .enhanced-input-file {
+            @include border-radius(4px 0 0 4px);
```

Comment:
I assume that all of the hardcoded sizes in this CSS are due to existing technical debt on this page. Which is sad because I thought this was a fairly new page.